README.md

```diff
@@ -21,12 +21,14 @@ Following inputs can be used as `step.with` keys
 | `name`      | String  | The name of the helm release |
 | `flags`      | String  | Additional flags to pass to Helm |
 | `chart-path`      | String  | The path to the chart. (defaults to `helm/`) |
+| `uninstall`      | String  | Whether to uninstall a Helm release. (defaults to `false`) |
+| `delete-namespace`      | String  | Whether to remove the namespace when uninstall a Helm release |
 
 
 ## Example usage
 
 ```yaml
-uses: craftech-io/eks-helm-deploy-action@v1
+uses: recurse-id/eks-helm-deploy-action@master
```

Comment:
should we move this to nirvana-health @recurse-id ?

Reply:
It must to be a public repository so we decided to host it here for security reasons.